src/components/Profile.js

```diff
@@ -1,16 +1,41 @@
 import React, { Component } from 'react';
 
 class Profile extends Component {
-
   render() {
+    const user = {
```

Comment:
I know I wrote this code before, but we could just get rid of the user object and just make the variables first level.